data/_ui/interfaces.txt

```diff
@@ -1515,6 +1520,11 @@ interface "info panel"
 
 
 
+interface "message log"
+	value "margin" 300
```

Comment:
Btw, why a margin instead of a width?

Reply:
I thought it would be more similar to the message log on the main panel (which is even wider):
```
box "messages"
	from 120 0 bottom left
	to -110 -200 bottom right
```

A fixed width sounds fine to me. I've tested some options and 720px width (compared to conversation panel's 570) looks good.

Reply:
I'd stick with a fixed width that's closer to some of the existing panels. The messages on the main panel might use a margin and be very wide, but those messages are text on an otherwise blank area. There's no UI except for the messages themselves. That's very different from the panel view that we're used to seeing in so many locations.